Formula/qt.rb

```diff
@@ -128,32 +128,31 @@ class Qt < Formula
 
   def install
     # Install python dependencies for QtWebEngine
-    venv = virtualenv_create(buildpath/"venv", "python3")
-    resources.each do |r|
-      venv.pip_install r
-    end
-    xy = Language::Python.major_minor_version "python3"
-    ENV.prepend_path "PYTHONPATH", buildpath/"venv/lib/python#{xy}/site-packages"
```

Comment:
QtWebEngine didn't build, so maybe this is still needed for some reason? No idea why though, given that the venv Python is first in `PATH`.

Also, it's weird that it just silently did not build.

Reply:
CMake build scripts are picking up `HOMEBREW_PREFIX/bin/python3`. In `CMakeCache.txt`:
```
//Path to a program.
Python3_EXECUTABLE:FILEPATH=/usr/local/bin/python3
```

Don't know why.

Not sure if there is a way to fail in this scenario. From https://github.com/qt/qtwebengine/blob/v6.3.1/src/CMakeLists.txt#L20-L36, it looks like related cache variables are going to be force modified based on upstream CI needing to pass configure step.

Reply:
I wonder if CMake is detecting the first `python[3]` it finds as belonging in a venv so skips it. That would explain why we need `PYTHONPATH`.